projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.html

```diff
@@ -3,89 +3,179 @@ <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
               </div>
-              <div class="info-card-image">
-                <span class="image-wrapper text-center">
-                  <cx-media
-                    class="image-item"
-                    [container]="quote.defaultCategory.image"
-                  ></cx-media>
-                </span>
-              </div>
-            </div>
-            <ng-container
-              *ngIf="
-                !quote.isExpired &&
-                quote.quoteWorkflowStatus.code !== 'REFERRED'
-              "
-            >
-              <div class="info-card-links">
-                <a
-                  class="link"
-                  [routerLink]="
-                    {
-                      cxRoute: 'quoteDetails',
-                      params: { quoteId: quote.quoteId }
-                    } | cxUrl
-                  "
+              <ng-container
+                *ngIf="
+                  !quote.isExpired &&
+                  quote.quoteWorkflowStatus.code !== 'REFERRED'
+                "
+              >
+                <div
+                  class="info-card-links justify-content-between align-items-center"
                 >
-                  {{ 'fscommon.details' | cxTranslate }}
-                </a>
-                <a class="link" (click)="retrieveQuote(quote)">
-                  {{ 'fscommon.retrieve' | cxTranslate }}
-                </a>
-              </div>
-            </ng-container>
+                  <div
+                    class="checkbox-wrapper"
+                    [class.invisible]="
+                      quotesByCategory[quote.defaultCategory.name]?.length < 2
+                    "
+                  >
+                    <div
+                      class="form-check d-flex align-items-center mb-0"
+                      [class.disabled]="
+                        quoteCodesForCompare?.length > 0 &&
+                        (selectedQuote?.defaultCategory?.code !==
+                          quote.defaultCategory.code ||
+                          (quoteCodesForCompare.length === 2 &&
+                            quoteCodesForCompare.indexOf(quote.cartCode) ===
+                              -1))
+                      "
+                    >
+                      <input
+                        #checkbox
+                        (click)="selectQuote(checkbox.checked, quote)"
+                        class="form-check-input"
+                        type="checkbox"
+                        value="{{ quote.cartCode }}"
+                        id="{{ quote.cartCode }}"
+                        [checked]="
+                          quoteCodesForCompare.includes(quote.cartCode)
+                        "
+                      />
+                      <label class="mb-0" for="{{ quote.cartCode }}">{{
+                        'quote.select' | cxTranslate
+                      }}</label>
+                    </div>
+                    <div
+                      class="tooltip-wrapper"
```

Comment:
Instead of this standard wrapper (`tooltip-wrapper`), just use `ml-3`. No need for this to be positioned absolutely in this case.